src/catalog/catalog_manager.py

```diff
@@ -48,7 +48,17 @@ def __init__(self):
         self._udf_service = UdfService()
         self._udf_io_service = UdfIOService()
 
-    def bootstrap_catalog(self):
+    def reset(self):
```

Comment:
Could you please check locally? reset function can not be called twice during my test.

Reply:
BTW, line 34-35 are redundant code? They are not used anywhere.

Reply:
Removed

Reply:
it now works on my system. Can you please verify?

Reply:
Verified